File: Arbeitsordner_Unity/Assets/Scripts/CheckApperiance.js
#pragma strict

function Start () {

}

function Update () {

}

function CheckForScale(scale: float, object:GameObject){
	
	// var tScale = object.transform;
	
	if (scale > 0) {
		object.transform.localScale = Vector3(scale, scale, scale);
	}
	Destroy(object.GetComponent(SphereCollider));
    object.AddComponent(SphereCollider);
}

File: Arbeitsordner_Unity/Assets/Scripts/CheckApperiance.cs
using UnityEngine;
using System.Collections;

 class CheckApperiance : MonoBehaviour {

	void  Start (){

	}

	void  Update (){

	}

	public void  CheckForScale ( float scale ,   GameObject obj  ){
		
		// Vector3 tScale= object.transform;

		if (scale > 0) {
			obj.transform.localScale = new Vector3(scale, scale, scale);
		}
		Destroy(obj.GetComponent<SphereCollider>());
		obj.AddComponent<SphereCollider>();
	}
}